### ISSUE TITLE
.NET 5 issue with PlatformAttribute

### ISSUE THREAD

**Issue by FixBo** [nunit/nunit#3565]
On .NET 5 Preview 4 exception occurred when using PlatformAttribute.

```
An exception was thrown while loading the test.
System.ArgumentException: Unknown framework version 5.5 (Parameter 'version')
   at NUnit.Framework.Internal.RuntimeFramework.ThrowInvalidFrameworkVersion(Version version) in D:\a\1\s\src\NUnitFramework\framework\Internal\RuntimeFramework.cs:line 235
   at NUnit.Framework.Internal.RuntimeFramework.InitFromFrameworkVersion(Version version) in D:\a\1\s\src\NUnitFramework\framework\Internal\RuntimeFramework.cs:line 226
   at NUnit.Framework.Internal.RuntimeFramework..ctor(RuntimeType runtime, Version version) in D:\a\1\s\src\NUnitFramework\framework\Internal\RuntimeFramework.cs:line 177
   at NUnit.Framework.Internal.RuntimeFramework.<>c.<.cctor>b__33_0() in D:\a\1\s\src\NUnitFramework\framework\Internal\RuntimeFramework.cs:line 145
   at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)--- End of stack trace from previous location
---
   at System.Lazy`1.CreateValue()   at System.Lazy`1.get_Value()
   at NUnit.Framework.Internal.RuntimeFramework.get_CurrentFramework() in D:\a\1\s\src\NUnitFramework\framework\Internal\RuntimeFramework.cs:line 261
   at NUnit.Framework.Internal.PlatformHelper..ctor() in D:\a\1\s\src\NUnitFramework\framework\Internal\PlatformHelper.cs:line 66
   at NUnit.Framework.PlatformAttribute..ctor(String platforms) in D:\a\1\s\src\NUnitFramework\framework\Attributes\PlatformAttribute.cs:line 37
```

**Comment by mikkelbu** [comment]
Thanks for reporting this @FixBo. I've edited your message to use a code block for better readability.

**Comment by AlexanderEgorov** [comment]
Also, the tests fixtures that has at least one test with **PlatformAttribute** are not discovered by NUnit and cannot be run.
I have attached a sample project in which it is impossible to run tests in the Tests1 class.
Would you please tell when the next update **3.13** will be published to NuGet?
[WpfTestsNet5.zip](https://github.com/nunit/nunit/files/5536616/WpfTestsNet5.zip)

**Comment by ChrisMaddock** [comment]
@AlexanderEgorov Just to clarify, are you saying that this issue is fixed in the current master, or is it outstanding? Can you file a new issue if not please, it sounds like a pretty big problem! 🙂 

On 3.13, the team were holding out for the official release of .NET 5. @rprouse is currently working on getting everything up and running in #3666 and I believe is planning to release after that.

**Comment by rprouse** [comment]
I've added this to the 3.13 release project. I'll download and look at your solution @AlekseyMartynov

**Comment by rprouse** [comment]
I've tested @AlexanderEgorov's example with a recent build of NUnit 3.13 and it works fine. Testing with 3.12, I see the same issue as reported in this issue.

The `Platform("NET")` attribute causes the one test to be ignored when using NUnit 3.13. That is correct since it is actually .NET Core, but I thought the code @jnm2 added would address that. We've only just kept the Platform attribute working and don't update it, so I've entered #3669 to decide how we should posisition it.
